### PR TITLE
[improve][website] Suggest pulsar-admin-client for Node

### PR DESCRIPTION
**Pull request by visortelle**
- [x] ‘doc’

### CODE REVIEW

File: site2/website/versioned_docs/version-2.9.2/client-libraries-node.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ All the methods in [producers](#producers), [consumers](#consumers), and [reader
 
 For 1.3.0 or later versions, [type definitions](https://github.com/apache/pulsar-client-node/blob/master/index.d.ts) used in TypeScript are available.
 
+If you are looking for Pulsar Admin API client, please try [pulsar-admin-client](https://github.com/tealtools/node-pulsar-admin-client).
```
Comment (Anonymitaet) [inline site2/website/versioned_docs/version-2.9.2/client-libraries-node.md:14]: @codelipenghui @hangc0276 can we add third-party tools here?

**Comment by Anonymitaet** [comment]
@visortelle https://docs.google.com/document/d/1Qw7LHQdXWBW9t2-r-A7QdFDBwmZh6ytB4guwMoXHqc0/edit
image

**Review by michaeljmarshall — CHANGES_REQUESTED** [review]
Thanks for your contribution @visortelle. We currently document third-party client libraries here in this file in the project `site2/docs/client-libraries.md`. You can update the versioned docs here `site2/website/versioned_docs/version-$VERSION/client-libraries.md`, assuming that your client is compatible with that version of Pulsar.

**Comment by tisonkun** [comment]
Closed as stale. Please rebase and resubmit the patch if it's still relevant.

**Pull request closed without merging**